### PR TITLE
data: vaibhav-xt (profile update)

### PR DESCRIPTION
**Pull request by vaibhav-xt**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.
- [x] This PR does not contain plagiarized content.

### CODE REVIEW

File: data/vaibhav-xt.json [hunk 1]
```diff
@@ -1,68 +1,138 @@
 {
   "name": "Vaibhav Maurya",
-  "bio": "Frontend Developer | Web3.0 Enthusiast | Content Creator | Teaching | Open 😺 Source Contributor | Gaming",
+  "type": "personal",
```
Comment (SaraJaoude) [inline data/vaibhav-xt.json:3]: this is no longer required:
```suggestion
```

File: data/vaibhav-xt.json [hunk 2]
```diff
@@ -1,68 +1,137 @@
 {
   "name": "Vaibhav Maurya",
-  "bio": "Frontend Developer | Web3.0 Enthusiast | Content Creator | Teaching | Open 😺 Source Contributor | Gaming",
+  "bio": "Web Developer 🌟 Open Source 🚀 | Freelance | Tech Blog @Hashnode📝 | Postman Student Expert 🌱 | Crafting digital dreams meticulously. Let's connect",
   "avatar": "https://github.com/vaibhav-xt.png",
+  "tags": [
```
Comment (SaraJaoude) [inline data/vaibhav-xt.json:5]: avatar is no longer required:

```suggestion
  "bio": "Web Developer 🌟 Open Source 🚀 | Freelance | Tech Blog @Hashnode📝 | Postman Student Expert 🌱 | Crafting digital dreams meticulously. Let's connect",
  "tags": [
```

**Review by SaraJaoude — APPROVED** [review]
I have removed the `type: personal` and `avatar` as these are no longer required - otherwise looks good and I will merge this PR.

**Comment by vaibhav-xt** [comment]
Thank you @SaraJaoude for reviewing and merging.

**Pull request merged**